integrations/huggingface/src/index.ts

```diff
@@ -0,0 +1,115 @@
+import { interfaces } from "@botpress/sdk";
+import { HfInference } from "@huggingface/inference";
+import * as bp from ".botpress";
+import axios from "axios";
+import { generateContent } from "./actions/generateContent";
+
+export default new bp.Integration({
+  register: async ({ ctx, client, logger }) => {
+    if (!ctx.configuration.languageModels) {
+      return;
+    }
+
+    const modelIds = ctx.configuration.languageModels
+      .split(",")
+      .map((el) => el.trim());
+
+    if (modelIds.length > 10) {
+      logger.forBot().debug(`Too many models provided`);
+      return;
+    }
+    const languageModels: interfaces.llm.Model[] = [];
+    for await (const id of modelIds) {
+      try {
+        const { data }: { data: { modelId: string; pipeline_tag: string } } =
+          await axios.get(`https://huggingface.co/api/models/${id}`);
```

Comment:
**nitpick:** I would personally extract this call in a dedicated function and a dedicated file. The reason is; this code violates the [single responsibility principle](https://en.wikipedia.org/wiki/Single-responsibility_principle).